tests/core/Jalasoft.TeamUp.Projects.Core.Tests/InvitationsServiceTests.cs

```diff
@@ -12,12 +12,15 @@
     public class InvitationsServiceTests
     {
         private readonly Mock<IInvitationsRepository> mockRepository;
+        private readonly Mock<IProjectsRepository> mockProjectsRepository;
         private readonly InvitationsService service;
 
         public InvitationsServiceTests()
         {
             this.mockRepository = new Mock<IInvitationsRepository>();
-            this.service = new InvitationsService(this.mockRepository.Object);
+            this.mockProjectsRepository = new Mock<IProjectsRepository>();
+
+            this.service = new InvitationsService(this.mockRepository.Object, this.mockProjectsRepository.Object);
```

Comment:
please could you add more unit tests here, since we added a logic in `UpdateInvitation` these two test cases come to my mind:
- make sure if the status is Accepted, the logic about updating the project shouldn't be hit
- in the other case, if Accepted the the logic about updating the project should be hit

Reply:
Done